grant-project/grant-monorepo
# Logos, Icons, Manifest.json

No associated issue. Still in active development due to ongoing design from Emily.

### What This Does

* Replaces all text logo instances of `Grant.io` with the wordmark logo.
* Adds [webapp-webpack-plugin](https://github.com/brunocodutra/webapp-webpack-plugin) to generate favicons and manifest.json for us.
    * Adds some server-side render logic to extract data from this to work with our render.
    * Favicon is temporary logo until we finalize.

The new plugin adds a decent amount of time to the initial compilation, and is a little hacked together to work for our SSR logic. I think we might want to investigate using the underlying module ([`favicons`](https://www.npmjs.com/package/favicons)) directly at some point.

### Steps to Test

* Build the site using the `logos-and-icons` branch
* Make sure the favicon shows up when you load the site
* Check the header for the correct logo, make sure it looks good on transparent and on white
* Check the footer for the correct logo
* Make sure the meta tags and manifest.json are correct

## Screenshots

### Homepage

<img width="1026" alt="screen shot 2018-10-29 at 3 35 01 pm" src="https://user-images.githubusercontent.com/649992/47676981-5b683980-db94-11e8-8a9f-f9aa09ea953c.png">

### Inner Page

<img width="948" alt="screen shot 2018-10-29 at 4 05 01 pm" src="https://user-images.githubusercontent.com/649992/47676999-66bb6500-db94-11e8-83ec-e307dedec733.png">

### Footer

<img width="475" alt="screen shot 2018-10-29 at 3 35 05 pm" src="https://user-images.githubusercontent.com/649992/47677006-6cb14600-db94-11e8-954c-fae6086afc25.png">
